openstudiocore/src/model/SpaceType.cpp

```diff
@@ -282,6 +280,26 @@ namespace detail {
   }
 
 
+  void SpaceType_Impl::parseStandardsJSON() const
+  {
+
+    if (m_standardsArr.empty()) {
+      // Embedded file path
+      std::string embedded_path = ":/Resources/standards/OpenStudio_Standards_space_types_merged.json";
+      std::string fileContent = ::openstudio::embedded_files::getFileAsString(embedded_path);
+
+      // Create a StandardsJSON
+      StandardsJSON standard(fileContent);
+
+      // Now try to get the primaryKey
+      std::string primaryKey = "space_types";
+
+      if (boost::optional<Json::Value> _standardsArr = standard.getPrimaryKey(primaryKey)) {
```

Comment:
should print a warning or error if not found right?

Reply:
That's not necessary until we implement the ability to use a supplied standards JSON. The primaryKey is harcoded there, and it'll be caught by the GTests if that stops working.